### PR TITLE
Testsuite: Clean up sle_ssh_tunnel_minion as well

### PR DESCRIPTION
**Pull request by calancha**
## Links
https://github.com/SUSE/spacewalk/issues/11427
(It fixes https://github.com/uyuni-project/uyuni/pull/2744)
### Ports
- Manager-4.1: https://github.com/SUSE/spacewalk/pull/12883
- Manager-4.0: https://github.com/SUSE/spacewalk/pull/12884
## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

### CODE REVIEW

**Review by Bischoff — APPROVED** [review]
+1

File: testsuite/features/step_definitions/salt_steps.rb [hunk 1]
```diff
@@ -721,7 +721,7 @@ def pillar_get(key, minion)
 
 And(/^I cleanup minion "([^"]*)"$/) do |minion|
   node = get_target(minion)
-  if minion == 'sle_minion'
+  if %w[sle_minion sle_ssh_tunnel_minion].include?(minion)
```
Comment (srbarrios) [inline testsuite/features/step_definitions/salt_steps.rb:724]:
```suggestion
  if minion.include? 'minion'
```

Comment (Bischoff) [reply testsuite/features/step_definitions/salt_steps.rb:724]: I think that's incorrect, this string comparaison will also catche ceos_minion etc.

Comment (srbarrios) [reply testsuite/features/step_definitions/salt_steps.rb:724]: I would make it more generic like that, so next time we add a new host name we don't need to touch this line

Comment (srbarrios) [reply testsuite/features/step_definitions/salt_steps.rb:724]: Well, and what's the problem to also match "ceos_minion"?
We are already using this construction in other steps, like in the "I apply highstate on"

Comment (srbarrios) [reply testsuite/features/step_definitions/salt_steps.rb:724]: I see, I just not read the next elseif... well, I would reconsider the other block, are we sure we can make a unique block?

Comment (calancha) [reply testsuite/features/step_definitions/salt_steps.rb:724]: No, that would include the second branch of the `if`.  We use `systemd` for Ubuntu/Centos minions.
See the full lines:
```
  if %w[sle_minion sle_ssh_tunnel_minion].include?(minion)
    node.run('rcsalt-minion stop')
    node.run('rm -Rf /var/cache/salt/minion')
  elsif %w[ceos_minion ceos_ssh_minion ubuntu_minion ubuntu_ssh_minion].include?(minion)
    node.run('systemctl stop salt-minion')
    node.run('rm -Rf /var/cache/salt/minion')
  end

```

**Pull request merged**